source/common/protobuf/utility.cc

```diff
@@ -884,13 +889,14 @@ void MessageUtil::redact(Protobuf::Message& message) {
 }
 
 ProtobufWkt::Value ValueUtil::loadFromYaml(const std::string& yaml) {
-  try {
-    return parseYamlNode(YAML::Load(yaml));
-  } catch (YAML::ParserException& e) {
+  TRY_NEEDS_AUDIT { return parseYamlNode(YAML::Load(yaml)); }
```

Comment:
@asraa @yanavlasov do we have any yaml parsing options that don't throw?

@chaoqin-li1123 do we really parse yaml on worker threads?

Reply:
No, it was on test thread, will fix it.